Formula/mongodb-community-shell.rb

```diff
@@ -4,8 +4,8 @@ class MongodbCommunityShell < Formula
 
   # frozen_string_literal: true
 
-  url "https://fastdl.mongodb.org/osx/mongodb-shell-macos-x86_64-4.2.0.tgz"
-  sha256 "68416de0b9652187ad2aa95c97ba31a031094c270975b23bc1f4ca22ee633360"
+  url "https://fastdl.mongodb.org/osx/mongodb-shell-macos-x86_64-5.0.0.tgz"
```

Comment:
Apparently we never updated this for 4.4?

Reply:
Apparently not!